tests/conftest.py

```diff
@@ -17,20 +17,33 @@ def is_responsive(url):
 
 
 @pytest.fixture(scope="session")
-def notebook_service(docker_ip, docker_services):
-    """Ensure that HTTP service is up and responsive."""
+def docker_compose(docker_services):
+    return docker_services._docker_compose
 
-    docker_compose = docker_services._docker_compose
 
-    # assurance for host user UID other that 1000
-    chown_command = "exec -T -u root aiidalab bash -c 'chown -R jovyan:users /home/jovyan/apps/aiidalab-widgets-base'"
-    docker_compose.execute(chown_command)
+def build_command(command, user=None):
```

Comment:
Does this need to be a separate function? Make more sense to me inside `aiidalab_exec` fixture.

Reply:
@unkcpz I think this function can now be moved to `aiidalab_exec` rather than being separete.